tools/downloader/converter.py

```diff
@@ -197,8 +252,25 @@ def convert(reporter, model):
     if failed_models:
         reporter.print('FAILED:')
         for failed_model_name in failed_models:
+            try:
+                telemetry.send_event('model_converter', 'failed', failed_model_name)
+            except NameError:
+                pass
             reporter.print(failed_model_name)
+        try:
+            telemetry.send_event('model_converter', 'result', 'fail')
```

Comment:
if you already send event per model, why do you need additional event with just status?
BTW, twhat the difference between send_error and send_evenl?

Reply:
My intention of sending the status was to collect statistics about failures and successes. Per-model statistics shows what models fails more.
The difference between `send_event` is that it sends events witch can be classified by category and actions, `send_error` basically sends error message, thought it isn't implemented yet https://github.com/openvinotoolkit/telemetry/blob/master/src/main.py#L84, probably we shouldn't use it.